packages/components/src/templates/next/components/internal/Vica/VicaWidgetClient.tsx

```diff
@@ -0,0 +1,46 @@
+"use client"
+
+import { useEffect } from "react"
+
+import type { VicaProps } from "~/interfaces"
+
+// Next.js pre-fetching caused widget to disappear on page navigation
+// Doing this forces the widget to load in between page navigation
+const loadVicaScript = () => {
+  const scriptTag = document.createElement("script")
+  scriptTag.async = true
+  scriptTag.type = "text/javascript"
+  scriptTag.src = "https://webchat.vica.gov.sg/static/js/chat.js"
+  scriptTag.referrerPolicy = "origin"
+  document.body.appendChild(scriptTag)
+}
+
+export const VicaWidgetClient = (vica: Omit<VicaProps, "site">) => {
```

Comment:
nit: I think this should be `VicaWidgetProps` instead of the `Omit`

Reply:
updated in https://github.com/opengovsg/isomer/pull/1086/commits/58e8143fd5e2a655a7ab62a826ca3d1b9aa1eeae